### ISSUE TITLE
Can't configure oncall with zvonok with helm

### ISSUE THREAD

**Issue by eugenefeel** [grafana/oncall#2520]
### What went wrong?

**What happened**:
  - Last week ago was added integration with service zvonok #2339 
  - If enable integration you will get different problems, because you didn't add service variables and you will can't use oncall. (blank interface)
![ScreenShot-2023-07-13](https://github.com/grafana/oncall/assets/139457476/cfd1ca57-427e-4651-b31d-cc53e277b52a)
![ScreenShot-2023-07-13](https://github.com/grafana/oncall/assets/139457476/af8464d1-8a64-4df1-87ef-ac8c4f6e5e30)

**What did you expect to happen**:
  - 1. @sreway you can rewrite your code without this error. If we haven't env, you can print about it in interface(Add exception)
  - 2. I will can create pull requests for update helm manifest

### How do we reproduce it?

1. Open Grafana OnCall and enable integration zvonok
2. Restart celery and engine
3. Wait for the celery to return traceback. Error message says: "│File "/etc/app/apps/phone_notifications/phone_provider.py", line 184, in get_phone_provider                                                                                                                │
│     return _providers[live_settings.PHONE_PROVIDER]                                                                                                                                                          │
│            ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                          │
│ KeyError: 'test'

### Grafana OnCall Version

v1.3.9

### Product Area

Alert Flow & Configuration, Helm

### Grafana OnCall Platform?

Kubernetes

**Comment by eugenefeel** [comment]
My colleagues found similar issue #2323

**Comment by sreway** [comment]
Hi, the issue arises when the environment variable PHONE_PROVIDER has an invalid or null value (default value is twilio), resulting in a KeyError exception [here](https://github.com/grafana/oncall/blob/71e77800ec24bd4c6a680bd28bdbe103b43c5e85/engine/apps/phone_notifications/phone_provider.py#L184)

**Comment by eugenefeel** [comment]
Yes. I reviewed code and uderstand it. I think that this wrong behavior. I could be wrong in PHONE_PROVIDER and what i'll do? I must have other way change this variable (with os.getenv) or maybe send me error about wrong phone_provider without save.

**Comment by sreway** [comment]
I made a PR that fixes this issue

**Comment by sreway** [comment]
Sorry, I don't have any ideas. When you add an environment variable in a Helm chart, it should be picked up by the deployed application if it is properly configured to read environment variables.

**Comment by eugenefeel** [comment]
hmm, okay. I'll try on weekend again. thx. Also we are waiting to approve the pull request :)

**Comment by eugenefeel** [comment]
UPD: If you change PHONE_PROVIDER in UI, it value will be add to tablespace base_livesetting and if you want change it with env you will can't. I think it's not normal, cos env need have high priority and if we change env this value coul'd be delete from this table. I resolve this problem with delete row in database, after my env was apply. @sreway what do u meen?

**Comment by sreway** [comment]
It seems to me that it is functioning correctly. If you want to reset the value to the default value set by the environment variable, you can click on the "Reset to default" button.

**Comment by eugenefeel** [comment]
Yes, you right but if we stay without value check we can't reset to default.

**Comment by eugenefeel** [comment]
Won't close issue cos i want configure helm for configure zvonok
